sources/HUBViewControllerImplementation.m

```diff
@@ -167,9 +165,23 @@ - (void)dealloc
 
 - (void)loadView
 {
-    self.view = [[HUBContainerView alloc] initWithFrame:CGRectZero];
+    HUBContainerView *containerView = [[HUBContainerView alloc] initWithFrame:CGRectZero];
 
-    [self createCollectionViewIfNeeded];
+    HUBCollectionView * const collectionView = [self.collectionViewFactory createCollectionView];
+    self.collectionView = collectionView;
+    collectionView.showsVerticalScrollIndicator = [self.scrollHandler shouldShowScrollIndicatorsInViewController:self];
+    collectionView.showsHorizontalScrollIndicator = collectionView.showsVerticalScrollIndicator;
+    collectionView.keyboardDismissMode = [self.scrollHandler keyboardDismissModeForViewController:self];
+    collectionView.decelerationRate = [self.scrollHandler scrollDecelerationRateForViewController:self];
+    collectionView.bounces = YES;
```

Comment:
The API isn't used by anything. `client-ios` uses `alwaysBounceVertical`, but I'll be moving that API into the view layer.